twitter.py

```diff
@@ -244,6 +245,15 @@ def handle_data(self, logs, data):
             logs.debug("Skipping tweet from user: %s (%s)" %
                        (screen_name, user_id_str))
             return
+        # Only look at POTUS tweets from POTUS, then check that tweet is signed -DJT
+        if user_id_str != POTUS_USER_ID:
```

Comment:
You are gonna need to add this to the original checker, because right now it won't even reach this part if it isnt by @realDonaldTrump